src/main/server.js

```diff
@@ -7,25 +7,28 @@ import { Scenario } from './modules/Scenario';
 import { Execution } from './modules/Execution';
 import { Output } from './modules/Output';
 
+const COLECTION_NAME = 'MSM-Scenario';
```

Comment:
My lonely consideration is that COLECTION_NAME shoud be COLLECTION_NAME. With double L

Reply:
Remember that we use camelCase for var names

Reply:
Samuel this is a constant, so for standard, it's better to have those in uppercase.

Reply:
Ok